From 4e9ea45803fe8dfc41ec5c51a65048addd985765 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Tue, 7 Jan 2020 16:44:38 +0100
Subject: [PATCH] tests/autopoint-3: unset MAKEFLAGS

This is needed when running ptests, as the MAKEFLAGS value (set up by run-ptest)
is messing up the test.

Upstream-Status: Inappropriate [oe-core specific]
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 gettext-tools/tests/autopoint-3 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gettext-tools/tests/autopoint-3 b/gettext-tools/tests/autopoint-3
index af2e17f..46cf3f7 100755
--- a/gettext-tools/tests/autopoint-3
+++ b/gettext-tools/tests/autopoint-3
@@ -149,6 +149,7 @@
     *) MAKE=make ;;
   esac
 fi
+unset MAKEFLAGS
 ${MAKE} >/dev/null 2>autopoint.err
 test $? = 0 || { cat autopoint.err; Exit 1; }
 
